src/langcheck/augment/en/_keyboard_typo.py

```diff
@@ -0,0 +1,47 @@
+from __future__ import annotations
+
+from nlpaug.augmenter.char.keyboard import KeyboardAug
+
+
+def keyboard_typo(
+    texts: list[str] | str,
```

Comment:
```suggestion
    texts: List[str] | str,
```
For consistency with other parts of the codebase?

Reply:
Using `typing.List` is deprecated by [PEP585](https://peps.python.org/pep-0585/#:~:text=Importing%20those%20from%20typing%20is%20deprecated.). I'll create an issue to handle them but should I restore these change for now?

Reply:
Ah good to here, then let's leave this line unchanged. Thanks for creating the issue for that!